identity/config.go

```diff
@@ -0,0 +1,20 @@
+package identity
+
+import "perun.network/go-perun/wallet"
+
+// WalletConfig defines wallet specific configuration parameters.
+type WalletConfig struct {
+	Addr         wallet.Address
```

Comment:
What address is this?

Reply:
`WalletConfig` is used to specify the config for accessing any account. It can onchain account or perun id account. This address corresponds to whichever address is being used. See `UserConfig` to see how it is used.

Reply:
I just wondered because there is normally more than one `Account`/`Address` per Wallet. Looking at `UserConfig` there seem to be at most two Accounts and they are assumed to be in different wallets. 
In the future it is planned to have one off-chain-`Account` per `Channel`, so this abstraction would not work.  
The `demo-node` only does it as simplification.  Im not sure though if we should rework it now or leave it as is.

Reply:
Thanks for pointing out. My idea was use one wallet for on-chain account and another one for off-chain IDs and participant IDs (As mentioned in the comment).
But I missed to add a variable to keep track of participant IDs. I have add another variable `PartAdds` to the user that keeps track of all participant IDs for all the channels the user is participating in.